src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -39,7 +39,27 @@
             </template>
             <div v-if="showParameters" id="parameters" class="form-group row">
                 <label for="params-component" class="col-sm-2 col-form-label text-right">Parameters</label>
-                <parameter-list id="params-component" @getParams="getParameterValues" :params="parameterValues"></parameter-list>
+                <parameter-list id="params-component" @getParams="getParameterValues"
+                                :params="parameterValues"></parameter-list>
+            </div>
+            <div v-if="showChangelog">
+                <div id="changelog-message" class="form-group row">
+                    <label for="changelogMessage" class="col-sm-2 col-form-label text-right">Changelog Message</label>
+                    <div class="col-sm-6">
+                        <textarea class="form-control" id="changelogMessage" v-model="changeLogMessageValue"
```

Comment:
It's currently possible to leave the changelog message blank even if you've selected a changelog type. I suggest that the message should be required if you select a type.

Reply:
...conversely it shouldn't be possible to add a message without specifying a type, as that will fail when it gets to orderly server. I wouldn't mind if the first changelog message type was auto-selected so there was always some type. But might be better as Mark says to require both to be populated if one is (it is ok to not include a changelog message at all).

We'd have to validate that before submitting, or enabling Run button. In that case, we'd probably want some message to the user to indicate missing information.

Reply:
Sorry - yes - this is a better solution given that new changelog types can't be added (in v1 at least). I guess we just need to ensure that empty changelog messages aren't sent in mrc-2161.